core/include/dt-bindings/clock/st,stm32mp25-rcc.h

```diff
@@ -0,0 +1,496 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) STMicroelectronics 2024 - All Rights Reserved
+ * Author: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
```

Comment:
As you did not put it in some .h, might as well remove it for consistency. We can always git blame